CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+# Contributing
+
+The development flow includes:
+
+- [create workspace, clone sources](#create-workspace-clone-sources)
+- [build](#build)
+- [run tests](#run-tests)
+- [generate devfile](#generate-devfile)
+- [create workspace](#create-workspace)
+- [package binaries](#package-binaries)
+- [push changes, provide pull request](#push-changes-provide-pull-request)
+
+## Create workspace, clone sources
+
+To create a workpace you can use [devfile](devfile.yaml):
+
+```
+$ chectl workspace:start -f https://raw.githubusercontent.com/che-incubator/chectl/master/devfile.yaml
+```
+
+> See more about [devfile](https://redhat-developer.github.io/devfile)
+
+After starting the workspace Theia will clone sources of `chectl` to `/projects/chectl` directory. Sources will be accessible in `theia-dev` and `dev` containers.
+
+#### 'theia-dev' container
+
+This container is used only for running Theia editor and does not take part in the following flow.
```

Comment:
I'm not sure to understand why we have a container that is not used ?
AFAIK we don't have theia-dev there

Reply:
theia-de, not theia-dev. It's my wrong.